src/main/java/address/controller/RootLayoutController.java

```diff
@@ -82,7 +75,7 @@ private FileChooser getXmlFileChooser() {
     @FXML
     private void handleNew() {
         logger.debug("Wiping current model data.");
-        PrefsManager.getInstance().clearSaveLocation();
+        modelManager.getPrefs().clearSaveLocation();
```

Comment:
Edits to prefs should go through a method in ModelManger, not through the prefs object (which should be read only)